### DISCUSSION TITLE
Rewrite issue template

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#87] [Ideas]
I plan to rewrite the issue templates.

I will start a new repository under org to do this.

When it's done, we push it [upstream](https://github.com/abbrcodes/abbreviations-in-code/tree/main/.github/ISSUE_TEMPLATE).

**Comment by oood** [comment]
@T1xx1 

I wish I could focus on this task these days.

I want all issues to start with "words" and have actions.

like: `Add: database` `Update: database`

I would then like to further format the templates for easy searching and even future automation.

Hopefully this thing doesn't conflict with the tasks you're about to do.

**Reply by T1xx1** [reply]
> @T1xx1 
> 
> I wish I could focus on this task these days.
> 
> I want all issues to start with "words" and have actions.

These actions will also help us in tasks so we can understand what's a task about only reading the title with the action.

> 
> like: `Add: database` `Update: database`

> I would then like to further format the templates for easy searching and even future automation.

Ok. 

> Hopefully this thing doesn't conflict with the tasks you're about to do.

Nop go ahead. This evening I'm doing the CONTRIBUTING issue.
